db/migrate/20220705162411_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] Your comments table uses `user_id`, whereas you have `author_id`. Kindly fix that by referencing `author`, instead of `user`. You can use the `to_table` option to refer Rails to the users' table.